src/Storageless/Http/SessionMiddleware.php

```diff
@@ -97,15 +85,15 @@ public function __construct(
     public static function fromSymmetricKeyDefaults(string $symmetricKey, int $expirationTime): self
```

Comment:
I propose to change the signature here to accept any [`Lcobucci\JWT\Signer\Key`](https://github.com/lcobucci/jwt/tree/master/src/Signer/Key)

Reply:
Not sure about this one: I guess it is better to expose `lcobucci/jwt` API here though, instead of requiring a `string`.

Reply:
So we agree, fine :+1: